### PR TITLE
Rework audit_deps to use full advisory IDs (Uplift to 1.32.x)

### PR DESCRIPTION
**Pull request by diracdeltas**
Fix https://github.com/brave/brave-browser/issues/19592
Uplift to 1.32.x

### CODE REVIEW

**Review by wknapik — APPROVED** [review]

**Comment by diracdeltas** [comment]
audit is failing due to https://github.com/advisories/GHSA-hqhp-5p83-hx96 - which should be ok to ignore but worth considering why it's different from master

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.32.x` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
